KsApi/Service+RequestHelpers.swift

```diff
@@ -51,10 +51,12 @@ extension Service {
       let task = URLSession.shared.dataTask(with: request) {  data, response, error in
         if let error = error {
           observer.send(error: .requestError(error, response))
+          print("🔴 [KsApi] Failure \(error.localizedDescription)")
```

Comment:
I know @justinswart has mentioned this in the Slack room but though I'll point it out again - there's this thing called 
`os_log` which defines various levels of logging which makes it more flexible to filter through in `Xcode.app` or `Console.app`. Logging network calls seems like a great example of `Debug` level logging.
 
https://medium.com/back-market-engineering/discovering-using-oslog-output-console-with-swiftybeaver-on-ios-54d7bfbe3b97

If you're not considering various level at this moment I'm sure `print` will do just fine!

Reply:
I was asking @Scollaco about this and he let me know `os_log` is for iOS 10 and above, so if I wanted to add it, I'd have to do `if #available` every place I log. So he recommended we use `print` for now and `os_log` when we update our app minimum to iOS 10. But I'm happy to add it if you feel strongly!

Reply:
We're still supporting iOS 9?

Reply:
I'm sorry Izzy..I had no idea we still support iOS 9

Reply:
@justinswart yeah 😬 

@dusi Nino said it's on the backlog to update our minimum OS 🚀